WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -92,7 +92,11 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
-    func shouldShow(for blog: Blog, apiResponse: BlogDashboardRemoteEntity? = nil) -> Bool {
+    func shouldShow(
+        for blog: Blog,
+        apiResponse: BlogDashboardRemoteEntity? = nil,
+        isJetpack: Bool = AppConfiguration.isJetpack
```

Comment:
Unfortunately, requiring callers to inject this triggered a cascade of other changes, so I left it with the default value.